src/lib/I18n.js

```diff
@@ -1,3 +1,5 @@
+/* global TRANSLATIONS */ // eslint-disable-line no-unused-vars
```

Comment:
This is required to add the dependency to the translations themselves. (Ironically, ESLint says it's an unused variable xD)